sktime/registry/tests/test_lookup.py

```diff
@@ -98,7 +98,10 @@ def test_all_tags(estimator_scitype):
 
     # there should be at least one tag returned
     # exception: scitypes which we know don't have tags associated
-    if estimator_scitype not in SCITYPES_WITHOUT_TAGS:
+    if estimator_scitype not in SCITYPES_WITHOUT_TAGS or (
+        isinstance(estimator_scitype, list)
+        and all(i not in SCITYPES_WITHOUT_TAGS for i in estimator_scitype)
+    ):
```

Comment:
wait, why can the scitype of an estimator be a list here?
Should it not always be a string, can you kindly explain how we could arrive at a list under your PR?

Reply:
It could be a List prior to my PR, this change is to stop it failing due to the test taking a list of scitypes all of which don't have tags.

The tests checks everything in ` estimator_scitype_fixture`, which is:
```
# selected examples of "search for two types at once to avoid quadratic scaling"
double_estimator_scitypes = [[b[i], b[(i + 3) % n]] for i in range(n)]
# fixtures search by individual scitypes, "None", and some pairs
estimator_scitype_fixture = [None] + BASE_CLASS_SCITYPE_LIST + double_estimator_scitypes
```

I think previously, when it was testing `double_estimator_scitypes` at least one would have tags. The inclusion of the early classifier base class has shifted the list a bit, allowing for the problem double scitype to appear in `double_estimator_scitypes`. At least that is what I have gathered from taking a look and attempting to fix it.

Reply:
ah, I see - because there could now be two scitypes that have no tags.
But, I thought, the early classifiers *do* have tags? So they should not be in the list withou tags?

Anyway, I agree that the test needs fixing, although my approach would be
```
est_list = estimator_scitype if isinstance(estimator_scitype, list) else [estimator_scitype]
if set(est_list).issubset(SCITYPES_WITHOUT_TAGS):
```
which seems less brittle to me

Reply:
I will try that.

Early classifiers do use tags, clusterers and I think series-annotators do not. Due to the way the doubles are selected, it so just happens that they weren't tested together previously.

Reply:
hm, that's not true?
I just looked into clusterers and randomly found `TimeSeriesKShapes` an `TimeSeriesKernelKMeans` using `capability:multivariate`

Reply:
I don't think the registry has clusterers listed as using it, for `capability:multivariate` only ["classifier", "early_classifier"] are listed. Clusterer is part of SCITYPES_WITHOUT_TAGS, current contents are: ["series-annotator", "clusterer"].

Reply:
well, that looks like an oversight then by whoever added that tag!
I would recommend to open an issue, or just fix it here, quickly.

Reply:
I would create an issue. Not really relevant to this PR, and may be a larger issue with how clustering interacts with tags currently.